rbhl/api.py

```diff
@@ -0,0 +1,166 @@
+from collections import defaultdict
+from decimal import Decimal
+from opal.core.views import json_response
+from opal.core.api import LoginRequiredViewset, episode_from_pk
+from rbhl.models import Demographics
+from opal.core.api import OPALRouter
+
+
+class PeakFlowGraphData(LoginRequiredViewset):
+    base_name = "peak_flow_graph_data"
+
+    def day_to_dict(self, peak_flow_day, demographics, pef):
+        aggregates = peak_flow_day.get_min_max_variability_completeness()
+
+        result = {
+            "note": peak_flow_day.note,
+            "treatment_taken": peak_flow_day.treatment_taken,
+            "day_num": peak_flow_day.day_num,
+            "date": peak_flow_day.date,
+            "work_day": peak_flow_day.work_day,
+            "pef_flow": pef
+        }
+
+        if not aggregates:
```

Comment:
Are we sure we don't want to give the front end the same datastructure but with `None` ?